vagrant/Vagrantfile

```diff
@@ -0,0 +1,126 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+  config.vm.box = "mrvantage/centos7-minikube"
+  config.vm.network "forwarded_port", guest: 3000, host: 3000
+  config.vm.synced_folder "..", "/vagrant"
```

Comment:
cf. the discussion on issue #396 there could be an issue with this.

Reply:
Thanks, @joel-bluedata https://github.com/bluek8s/kubedirector/pull/403/commits/a31a6aab2923611f3a8e5f0f1ba9c0d92a2d1dca fixes #396